commands/COVID/covid-stats.js

```diff
@@ -49,31 +45,36 @@ module.exports = {
 						return;
 					}
 					// display covid stats for one country
-					const filterTerm = args.join(' ');
+					const filterTerm = args.join(" ");
 					const stats = countryStats(filterTerm, data);
 
 					if (!stats) {
 						const noargembed = new Discord.MessageEmbed()
-						.setTitle(":octagonal_sign:  No Argument Given  :octagonal_sign:")
-						.setColor(colors.heptagram)
-						//const { prefix, token, roles, MongoDB, serverId, colors } = require('../../config.json');
-						.setDescription(`Oh no! You didn’t specify a country. Type \`${prefix}covid-stats list\` for the list of available countries`)
-					message.channel.send(noargembed);
+							.setTitle(
+								":octagonal_sign:  No Argument Given  :octagonal_sign:"
+							)
+							.setColor(colors.heptagram)
+							//const { prefix, token, roles, MongoDB, serverId, colors } = require('../../config.json');
```

Comment:
```suggestion
							const { prefix, token, roles, MongoDB, serverId, colors } = require('../../config.json');
```

Reply:
The commented code was there when pulled. It has to be removed or stay commented as it is located between chained method calls of a function. Would you like me to remove it?

Reply:
yes please!